repository-data/application/src/main/resources/hcm-config/hst/hosts/br-cloud/cloud.yaml

```diff
@@ -1,18 +1,17 @@
 definitions:
   config:
-    /hst:platform/hst:hosts/prd-brcloud/cloud:
+    /hst:platform/hst:hosts/br-cloud/cloud:
       jcr:primaryType: hst:virtualhost
       hst:scheme: https
       hst:showcontextpath: true
       hst:showport: false
       /bloomreach:
         jcr:primaryType: hst:virtualhost
-        /hee:
+        /${brc.namespace}:
           jcr:primaryType: hst:virtualhost
-          /cms-prod:
+          /${brc.environmentname}:
             jcr:primaryType: hst:virtualhost
             /hst:root:
               jcr:primaryType: hst:mount
               hst:ismapped: false
               hst:namedpipeline: WebApplicationInvokingPipeline
-              hst:alias: cms-mount
```

Comment:
I think we need this alias as it has been referred in BlogPostCommentMailFormDataBehavior.java